normalizer/app/geo_processor.py

```
@@ -35,131 +39,190 @@ def replacer(match):
-def render_template(template_str: str, row: List[str], cache: Dict[str, Any], 
-                   full_api_address: str, api_key: str, sleep_msec: int) -> str:
+    params = {"address": address, "key": api_key, "language": "ja"}
+    try:
+        res = requests.get(url, params=params)
+        data = res.json()
+        status = data.get("status")
+        if status == "OK":
+            loc = data["results"][0]["geometry"]["location"]
+            return loc["lat"], loc["lng"]
+        else:
+            return None, None
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)